### ISSUE TITLE
Connect to `127.0.0.1` instead of `localhost`

### ISSUE THREAD

**Issue by mxsdev** [stateful/vscode-runme#456]
Would be nice to connect directly to `127.0.0.1` and use this as the server address, rather than `localhost`, since `localhost` is technically not guaranteed to exist. This has already caused issues for a user in #447 

Trying to do this right now results in the following console message:

![image](https://user-images.githubusercontent.com/16108792/234700704-f554c2ec-bd0d-4a8b-a74e-8d59617aa93b.png)

Functionality still works though.

**Comment by sourishkrout** [comment]
Perhaps we can attempt connecting to `localhost` for several successive tries before switching to `127.0.0.1` to circumvent this ugly warning until Electron moves to node 18. Wdyt @mxsdev?

**Comment by mxsdev** [comment]
@sourishkrout 

Yeah, that could work. 

To be honest I'd personally rather just eat the warning or keep it as `localhost` than do that. Because it has to work nicely with our existing retry mechanism...But I'm fine with doing that if you really want

> to circumvent this ugly warning until Electron moves to node 18. 

Are we blocked on upgrading `grpc-node` because of this? (If so I didn't know that)

**Comment by sourishkrout** [comment]
Not a priority. Let's keep this in the backlog for now.

> > to circumvent this ugly warning until Electron moves to node 18.
> 
> Are we blocked on upgrading `grpc-node` because of this? (If so I didn't know that)

I thought the error came out of opensll linked with `node 16` but could also `be grpc-node` 🤷.
